action.yml

```diff
@@ -45,6 +45,12 @@ inputs:
   tags:
     description: "List of tags"
     required: false
+  load_target:
+    description: "Target stage to load for local use"
+    required: false
```

Comment:
we have these marked as required ones in our workflows, but since they may not be in other repositories' build actions, they are marked required: false.
CC @vinayteki95